src/ts/ssh-tcp/services/portForwardingService.ts

```diff
@@ -61,8 +61,25 @@ export class PortForwardingService extends SshService {
 	public static readonly portForwardChannelType = 'forwarded-tcpip';
 	public static readonly reversePortForwardChannelType = 'direct-tcpip';
 
+	/**
+	 * Maps from local IP address and FORWARDED port number to the object that manages
+	 * listening for incoming connections on that port and forwarding them through the session.
+	 *
+	 * Note the actual local source port number used may be different from the forwarded port
+	 * number if the local TCP listener factory chose a different port. The forwarded port number
+	 * is used to identify the port in any messages exchanged between client and server.
+	 */
 	private readonly localForwarders = new Map<string, LocalPortForwarder>();
-	private readonly remoteConnectors = new Map<string, RemotePortConnector>();
+
+	/**
+	 * Maps from FORWARDED port numbers to the object that manages relaying forwarded connections
+	 * from the session to a local port.
+	 *
+	 * Note the actual local destination port number used may be different from the forwarded port
+	 * number. The forwarded port number is used to identify the port in any messages exchanged
+	 * between client and server.
+	 */
+	private readonly remoteConnectors = new Map<number, RemotePortConnector>();
```

Comment:
Is `PortForwardingService.cs` already fixed? I don't see it in the PR.

Reply:
It is in the PR.

Reply:
Found it